src/xscen/biasadjust.py

```diff
@@ -277,13 +283,21 @@ def adjust(
         dtrain.attrs["train_params"] = eval(dtrain.attrs["train_params"])  # noqa: S307
 
     var = dtrain.attrs["train_params"]["var"]
-    if len(var) != 1:
-        raise ValueError(
-            "biasadjust currently does not support entries with multiple variables."
-        )
-    else:
+    if len(var) == 1:
         var = var[0]
         sim = dsim[var]
+    else:
+        sim = xsdba.stack_variables(dsim[var])
+
+    # Used in MBCn adjusting
+    # I'm just assuming that if `ref` is needed, so is `hist`
+    if "ref" in xsdba_adjust_args:
+        train_period = dtrain.attrs["train_params"]["period"]
+        ref = xsdba.stack_variables((xsdba_adjust_args["ref"])[var])
+        ref = ref.sel(time=slice(train_period[0], train_period[1]))
+        hist = sim.sel(time=slice(train_period[0], train_period[1]))
```

Comment:
fun fact, tu peux faire:
```suggestion
        ref = ref.sel(time=slice(*train_period))
        hist = sim.sel(time=slice(*train_period))
```

Reply:
Ouais je préfère ça aussi! je le change merci